src/zck.c

```diff
@@ -55,6 +55,8 @@ static struct argp_option options[] = {
      "Set zstd compression dictionary to FILE"},
     {"manual-chunk",       'm', 0,           0,
      "Don't do any automatic chunking (implies -s)"},
+    {"chunk-hash-type",     'h', "HASH",     0,
+     "Set hash type to one of sha1, sha256, sha512, sha512_128"},
```

Comment:
Let's leave sha1 out of this list.  We need to support it in the library because we guarantee backwards compatibility, but I see no reason to have it here.